doc/source/conf.py

```diff
@@ -33,7 +33,7 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 root_dir = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
```

Comment:
I would use the `inspect` syntax also here:
```suggestion
root_dir = Path(inspect.getfile(lambda: None)).parents[2]
```

Reply:
But do we require the inspect construct here?

Reply:
I remember the `__file__` syntax is somehow weaker than the one with `inspect`, see e.g. PR #102.

Reply:
Ah, I thought this had to do with calling cobrawap from CLI, that's why I asked. I think here, both versions work, either is fine with me.